### ISSUE TITLE
Cannot run with sound null safety.

### ISSUE THREAD

**Issue by inc16sec** [gql-dart/gql#213]
After migrating to null-safety and updating all packages I wanted to run the App and this error showed up.

```
Error output from Xcode build:
↳
    ** BUILD FAILED **
Xcode's output:
↳
    Error: Cannot run with sound null safety, because the following dependencies
    don't support null safety:
- package:gql
    For solutions, see https://dart.dev/go/unsound-null-safety
    ../../../../documents/flutter/.pub-cache/hosted/pub.dartlang.org/graphql_flutter-5.0.0-nullsafety.1/lib/src/widgets/query.dart:62:44: Warning: Operand of null-aware operation '!' has type 'DefaultPolicies' which excludes null.
     - 'DefaultPolicies' is from 'package:graphql/src/core/policies.dart' ('../../../../documents/flutter/.pub-cache/hosted/pub.dartlang.org/graphql-5.0.0-nullsafety.2/lib/src/core/policies.dart').
        optionsWithOverrides.policies = client.defaultPolicies!.watchQuery

```

I've tried to run the app with no sound null safety using `flutter run --no-sound-null-safety` yet I still run to some issues because the project now relies on null-safety.
Is this an issue on my part or there are packages that still need to be updated.
Thank you.

**Comment by agent3bood** [comment]
Wrong repository, you issue is related to this package https://github.com/zino-app/graphql-flutter

**Comment by inc16sec** [comment]
I see, sorry for the inconvenience.

**Comment by inc16sec** [comment]
I fixed it. 
I was actually overriding `graphql_flutter: 5.0.0-nullsafety.1` and `gql:  ^0.12.4` so I can use the package Artemis. And it was  `gql` that was causing the app not to run, as for `graphql_flutter` it was showing me warnings and they went away as soon as I changed `graphql_flutter: 5.0.0-nullsafety.1` to `graphql_flutter: 5.0.0-nullsafety.2` in dependency_overrides.
like this.

```yaml
dependency_overrides:
 graphql_flutter: 5.0.0-nullsafety.1
```
to
```yaml
dependency_overrides:
 graphql_flutter: 5.0.0-nullsafety.2
```
and of course I had to also change `gql` from `gql:  ^0.12.4` to a null-safe version in my case it was
```yaml
dependency_overrides:
gql: ^0.13.0-nullsafety.2 
 graphql_flutter: 5.0.0-nullsafety.2
```
And that's it now the app is running just fine.
